cfme/tests/services/test_provision_stack.py

```diff
@@ -140,10 +140,15 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.fixture(scope="function")
-def dialog(provisioning):
-    dialog_name = "dialog_" + fauxfactory.gen_alphanumeric()
+def dialog(provider, provisioning):
     template_type = provisioning['stack_provisioning']['template_type']
-    template = OrchestrationTemplate(template_type=template_type,
+    if provider.type == "azure":
```

Comment:
Azure comes with a pre built template and dialog. so Using that in the test. 
Where as for other providers , we need to add a new template